public/content/en/basics/alias-strings.md

```diff
@@ -72,10 +98,12 @@ void main() {
     string str = format("%s %s", "Hellö",
         "Wörld");
```

Comment:
Should we pick a better example that also shows different values for all metrics (like your inline example)?